tasks/lib/run.js

```diff
@@ -104,9 +105,79 @@ RunUtil.displayScriptOutput = function(grunt, outputs, data) {
   }
 };
 
+function getConstructorName(value) {
+  if(value == null) { // is it undefined or null
+    return String(value);
+  }
+
+  return value.constructor.name;
+}
+
+function getFallbackErrorMessage(err) {
+  switch(err.__fxotype__) {
+    case 'AuthError':
+      return 'The request failed because of an authorization problem.';
+    case 'ServiceError':
+      return 'There was an error with your task, please contact support.';
+    case 'WebhookIgnoreError':
+      return 'The data received by the webhook was ignored.';
```

Comment:
The core is going to treat ignored webhooks differently - it will silently ignore them, meaning that nothing will appear in the UI logger modal. Is there a way that this could be mimicked in the SDK? Ideally the webhook would be silently ignored, and the SDK would continue waiting for a non-ignored webhook.